docs/common/widget.ldoc

```diff
@@ -1,11 +1,11 @@
 
---- Get a widex index.
+--- Get a wibox index.
```

Comment:
Shouldn't this be `widget` instead of `wibox`? It is in the docs for a widget and the `:index` function (AFAIR) is supposed to say something "the widget you gave me is my fourth entry".